src/runtime_src/core/pcie/driver/linux/xocl/subdev/xdma.c

```diff
@@ -66,19 +66,28 @@ static ssize_t xdma_migrate_bo(struct platform_device *pdev,
 	int nents = sgt->orig_nents;
 	pid_t pid = current->pid;
 	int i = 0;
-	ssize_t ret;
+	ssize_t ret = 0;
+	ssize_t migrated = 0;
 	unsigned long long pgaddr;
 
 	xdma = platform_get_drvdata(pdev);
 	xocl_dbg(&pdev->dev, "TID %d, Channel:%d, Offset: 0x%llx, Dir: %d",
 		pid, channel, paddr, dir);
-	ret = xdma_xfer_fastpath(xdma->dma_handle, channel, dir,
+	migrated = xdma_xfer_fastpath(xdma->dma_handle, channel, dir,
 		paddr, sgt, false, 10000);
-	if (ret >= 0) {
+	if (migrated >= 0) {
 		xdma->channel_usage[dir][channel] += ret;
```

Comment:
I would not recommend making this type of change so close to a release. Or maybe you could make sure you can test qdma and xdma carefully to make sure you did not introduce any regression. E.g., here you probably failed to update channel_usage correctly.